src/components/top-stories/components/top-stories-list/TopStoriesList.tsx

```diff
@@ -10,11 +11,14 @@ type Props = {
 export const TopStoriesList: FC<Props> = ({ topStories }) => {
   return (
     <ul className="flex flex-col gap-y-5">
-      {topStories.map((item, index) => (
-        <Link href={''} key={`${index}-${item.title}`}>
-          <TopStoriesCard topStory={item} />
-        </Link>
-      ))}
+      {topStories.map((item, index) => {
+        const slug = generateSlug(item.title);
+        return (
+          <Link href={`/top-stories/${slug}`} key={`${index}-${item.title}`}>
```

Comment:
it's a good practise to have a builder of all pages in application. So in case if you need to change something in route - you change in one place.

in constants/pages.ts
```
export const DYNAMIC_PAGES = {
  getTopStoriesPage: (slug: string) => `/top-stories/${slug}`,
};

```